src/lib.rs

```diff
@@ -4,6 +4,9 @@
 //! implemented in the [category](category/index.html), [keyword](keyword/index.html),
 //! [krate](krate/index.html), [user](user/index.html) and [version](version/index.html) modules.
 
+// <https://doc.rust-lang.org/rustdoc/lints.html#unescaped_backticks>
+#![warn(rustdoc::unescaped_backticks)]
+
```

Comment:
can we enable this via something like https://github.com/rust-lang/crates.io/blob/c847b96cc64af63f57e068d8971b606cce86a6ad/Cargo.toml#L22-L24 so that it's applied to all crates in the workspace?

Reply:
Ah definitely! I didn't notice the workspace config :+1:

Reply:
sweeet! thank you